content/docs/0.16.x/manage/service/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Service
+title: Deploying Services
```

Comment:
Unrelated

Reply:
Bonus points for using separate pull requests next time. Or at least mentioning  such refactorings in the PR title